ios/RNAudioRecorderPlayer.swift

```diff
@@ -229,6 +231,8 @@ class RNAudioRecorderPlayer: RCTEventEmitter, AVAudioRecorderDelegate {
                 AVFormatIDKey: avFormat!,
                 AVNumberOfChannelsKey: numberOfChannel!,
                 AVEncoderAudioQualityKey: audioQuality!,
+                AVEncoderBitRateKey: bitRate!,
+                AVEncoderBitRatePerChannelKey: bitRatePerChannel!,
```

Comment:
What would be the default value here when the variables are not passed?

Reply:
@hyochan I'm not really much of a swift dev (I know enough to have made that patch for the project I'm working on) but whatever they were before aka undefined?

Reply:
@moulie415 Thanks for coming back! Well I think since the variable is `bitRate!`, I think the app will throw the error when it is `undefined`. Could you kindly test it out so that we won't suffer from the regression?